utils/compare/README.md

```diff
@@ -0,0 +1,99 @@
+# Compare 
+
+Be able to compare a current runs data with all previous runs data based on matching: 
+
+* benchmark type
+* Worker node type
+* Master node type
+* network type
+* Platform
+* Version 
+* Only Successful jobs
+
+## Enviornent variables set
+
+* previous_version: Set to False by default, can compare with previous version 
+* tolerancy_num: Set to 20 be default, meaning if current uuid is worse than aggregated data by 20%, Negative values mean uuid/current run is better 
+* UUID: current UUID to use as comparison against
+
+## Steps: 
+* Get metadata of current run using enviornment variable UUID and matching data points above
+* * Get data from: index=perf_scale_ci
+* For all matching uuids get aggregated data points based on metrics specified in configs
+* Combine current uuid data with aggregated comparison uuid data
+* Generate percentage difference in those data points
+* Based on tolerancy enviornment variable, decide to pass or fail each of the comparison points
+
+
+## Output
+<metrics_from_config_file>,<additionalColumns_from_config_file>,
+<uuid>_value,<comparison_uuid(s)>_value,% difference,result
+
+
+Kube Burner example output
+```
+uuid - value_x: 35e86556-6375-466b-8342-65a85170fe3e
+comparison ids - value_y: ['8eb2fb3c-a5a0-4016-a526-dab0913ca97e']
+,metricName,quantileName,P99_x,P99_y,difference,result
+0,podLatencyQuantilesMeasurement,ContainersReady,4.0,4.0,0.0,Pass
+1,podLatencyQuantilesMeasurement,Initialized,1.0,1.0,0.0,Pass
+2,podLatencyQuantilesMeasurement,PodScheduled,1.0,0.0,-100.0,Fail
+3,podLatencyQuantilesMeasurement,Ready,4.0,4.0,0.0,Pass
+```
+
+
+## How to Run
+
+```
+pip3 install -r utils/compare/requirements.txt
+export UUID=<uuid>
+export ES_USERNAME=<es_username>
+export ES_PASSWORD=<es_password>
+
+python3 utils/compare/read_files.py
+```
+
+
+### Optional BASELINE_UUID
+
+If you know what uuid or set of uuids you want to compare to set the enviornment variable BASELINE_UUID with a comma separated string list
+
+## Network Perf V2
+
+Config file: "configs/k8s.json
+Index = "k8s-netperf"
+
+## Ingress Perf
+index = "ingress-performance"
+Config file: configs/ingress.json
+
+## Kube Burner 
+
+index = "ripsaw-kube-burner"
+
+* configs/podlatency.json
+* configs/crio.json
+* configs/kubelet.json
+* configs/etcd.json
+* configs/master_node.json
+* configs/worker_node.json*
+
+## Defining new config
+
+Each config is a json file consisting of 
+* metrics: A list of dictionairies each contiaining a list of the key values you want to compare upon (can use regex on some values, depends on set up in elasticsearch). Multiple items in this list will still use the same below fields with every search in addition to the key/values set in the dictionary
```

Comment:
Can we specify those `same below fields` used in addition to the key/values set in the dictionary, please?